shardingsphere-test/shardingsphere-pipeline-test/src/test/resources/scaling/rule_alter/source_rules_config.yaml

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- !SHARDING
+  defaultDatabaseStrategy:
+    standard:
+      shardingAlgorithmName: database_inline
+      shardingColumn: user_id
+  defaultTableStrategy:
+    none: ''
+  keyGenerators:
+    snowflake:
+      type: SNOWFLAKE
+  scaling:
+    default_scaling:
+      completionDetector:
+        props:
+          incremental-task-idle-minute-threshold: 30
+        type: IDLE
+      dataConsistencyChecker:
+        props:
+          chunk-size: 1000
+        type: DATA_MATCH
+      input:
+        batchSize: 1000
+        workerThread: 4
+      output:
+        batchSize: 1000
+        workerThread: 4
+      streamChannel:
+        props:
+          block-queue-size: 10000
+        type: MEMORY
```

Comment:
`input/output/streamChannel` is not necessary to configure, we could just keep the required configuration.